src/App.jsx

```diff
@@ -1,36 +1,21 @@
 import { useState } from "react";
-
-// import "./App.css";
-var todoList = [
-  {
-    id: 1,
-    title: "Assignment not start",
-  },
-  {
-    id: 2,
-    title: "Assignment start",
-  },
-  {
-    id: 3,
-    title: "Assignment completed",
-  },
-];
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
 
 function App() {
+  const [todoList, setTodoList] = useState([]);
+
+  function addTodo(newTodo) {
+    setTodoList((previousTodoList) => [...previousTodoList, newTodo]);
```

Comment:
Good job here man,

if you wanted to make this shorter and more concise, you could do this as well:

`setTodoList([...todoList, newTodo]);`